src/pages/Signup.jsx

```diff
@@ -0,0 +1,300 @@
+// React Imports
+import React, { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { TextField, Button, CardHeader } from '@mui/material';
+
```

Comment:
We should probably replace empty lines with relevant labels such as `// Material UI Imports` and the same for hooks and contexts. Also arrange them the same way that we have with other files (e.g. group all MUI imports together and place everything in the same descending order)

Although now I'm noticing a similar case in `Profile.jsx` and perhaps elsewhere

Reply:
fixed